go/chat/boxer.go

```diff
@@ -219,6 +222,10 @@ func (b *Boxer) boxMessageV1(ctx context.Context, msg chat1.MessagePlaintextV1,
 		}
 	}
 
+	if len(msg.ClientHeader.TlfName) == 0 {
```

Comment:
Actually on a second thought, this should never happen as long as the RPC call to kbfs works as expected. Otherwise, it would just either fail at the `len(tlfName)` check above, or one of the error checks for kbfs rpc calls.

Did you see this happening?